### DISCUSSION TITLE
[2-1] 클래스 내부에 정의되지 않은 함수에 관하여

### DISCUSSION THREAD

**Discussion by moonuckjoo** [bbang-jun/2023-OOP-Mentoring#7] [Q&A]
public:
	Student_info(char* name, int age, char* univ, char* major);
	bool find(char* name);
	void print();
	void change(char* name, int age, char* univ, char* major);
};
예를 들면 이런 클래스 내부의 퍼블릭에서 print메소드는 정의되지 않았으니 따로 정의해줘야 동작하는 건가요?
public:
	Student_info(char* name, int age, char* univ, char* major);
	bool find(char* name);
	void print(){cout << ~~~~};
	void change(char* name, int age, char* univ, char* major);
};
이런 식으로요!

**Comment by bbang-jun** [comment]
![image](https://user-images.githubusercontent.com/90714030/215461926-76ee196a-af36-47f7-b793-0831363ada84.png)

Student_info 멤버 이미지와 똑같이 코드를 작성하셔야 합니다. 

해당 이미지를 보시면 메서드들이 선언만 되어 있는 것을 확인하실 수 있습니다.

따라서 제가 알려드린 방법인 **선언 및 정의** 방식으로 함수를 작성하셔야 합니다.

즉, 해당 class 내부에는 선언만 하시고 class 외부에서 메서드를 정의(구현)하시면 됩니다.
